helix-core/src/test/conf/testng.xml

```diff
@@ -18,7 +18,7 @@
   ~ under the License.
   -->
 <!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd">
-<suite name="Suite" time-out="300000">
+<suite name="Suite" time-out="900000">
```

Comment:
Does any test definitely need more than 5 minutes?

Reply:
At least several. ZK in github sometimes is slow.

Reply:
TestEnableCompression for example.